README.md

```diff
@@ -1,3 +1,5 @@
+/*
```

Comment:
My `tango` crate that does this. You can see a [demo here][tango-demo]. (There's other expository material elsewhere, but I suspect it will do more harm than good for me to list all that stuff here.)

[tango-demo]: https://github.com/pnkfelix/tango-demo/